beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/context/CommonBeans.java

```diff
@@ -47,32 +47,39 @@
 @EnableRetry(proxyTargetClass = true)
 public class CommonBeans {
 
-  @Value("${properties.apiary.queue-url}")
-  private String queueUrl;
+  @Autowired private List<SchedulerService> schedulerServices;
```

Comment:
Are these fields needed? If they are removed wont Spring just figure out what to inject from the method parameters?

Reply:
nope! good catch, removed.